Sources/Build/describe().swift

```diff
@@ -30,7 +30,7 @@ public func describe(_ prefix: AbsolutePath, _ conf: Configuration, _ modules: [
     let Xcc = flags.cCompilerFlags.flatMap{ ["-Xcc", $0] }
     let Xld = flags.linkerFlags.flatMap{ ["-Xlinker", $0] }
     let prefix = prefix.appending(component: conf.dirname)
-    try Utility.makeDirectories(prefix.asString)
+    try mkdir(prefix)
```

Comment:
I actually picked `makeDirectories` intentionally at some point. My motivation was that I find it very confusing to have functions with POSIX names but not POSIX semantics in a codebase which _also_ uses POSIX in places.